### PR TITLE
Update caption

### PR DESCRIPTION
**Pull request by BinduTuli0609**
Updates the caption in BinduTuli0609.md file

### CODE REVIEW

File: _posts/0000-01-05-BinduTuli0609.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Is the training going too slow or are we on track?
```
Comment (NowshadAzimi) [inline _posts/0000-01-05-BinduTuli0609.md]: Hmm, that's a question for the training guides.

File: _posts/0000-01-05-BinduTuli0609.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (NowshadAzimi) [inline _posts/0000-01-05-BinduTuli0609.md]: Perhaps, change the title
```suggestion
title: "Welcome to Bindu's deck!"
```

File: _posts/0000-01-05-BinduTuli0609.md (hunk 1 above)
Comment (beardofedu) [reply _posts/0000-01-05-BinduTuli0609.md]: It is a little slower 🐢 but we aren't in danger 💀 of missing any content 😺

File: _posts/0000-01-05-BinduTuli0609.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to Bindu's deck!"
```
Comment (BinduTuli0609) [inline _posts/0000-01-05-BinduTuli0609.md]: Accept

**Pull request merged**